apps/emqx/src/emqx_ds_schema.erl

```diff
@@ -39,6 +46,14 @@
 %% API
 %%================================================================================
 
+add_handler() ->
+    A = '?',
+    [
+        ok = emqx_config_handler:add_handler([namespace() | L], ?MODULE)
+     || L <- [[A], [A, A], [A, A, A], [A, A, A, A]]
```

Comment:
It is very weird that I needed this, I wonder if there's a better way to get updates for the entire tree.